dev-env.sh

```diff
@@ -57,7 +57,7 @@ function workspace-kill-port {
     echo "There are no processes listening to the port $port."
   else
     echo "Killing the processes listening to the port $port."
-    kill $pids
+    kill -9 $pids
```

Comment:
As a suggestion, we could try killing normally and then force killing if the port is still in use.

function workspace-kill-port {
  local port="$1"
  
  # Check if the port exists
  if ! lsof -i :$port > /dev/null; then
    echo "Port $port is not open."
    return
  fi
  
  # Get PIDs of processes listening on the port
  local pids=$(lsof -t -i:$port)
  
  if [ -z "$pids" ]; then
    echo "No processes found listening on port $port."
    return
  fi
  
  echo "Attempting to kill processes listening on port $port..."
  
  # First, try a regular kill
  kill $pids
  
  # If the above fails, use kill -9
  if ! kill -0 $pids > /dev/null 2>&1; then
    echo "Failed to kill processes. Using SIGKILL"
    kill -9 $pids
  fi
  
  # Verify if the processes were killed successfully
  if ! ps -p $pids > /dev/null; then
    echo "Processes listening on port $port have been successfully terminated."
  else
    echo "Warning: Processes listening on port $port were not terminated successfully."
  fi
}